app/views/posts/show.html.haml

```diff
@@ -21,7 +21,14 @@
                 %p= @post.body
                 - if @post.image.attached?
                   = image_tag url_for(@post.image), alt: 'Post Image', class: 'post-image'
-            %i.bi.bi-chat-right.toggle-postcomment-form{id: "toggle-postcomment-form-#{@post.id}", data: { post_id: @post.id }}
+            .container.d-flex.post-icons-container
+              .like-unlike-container{id: "like_unlike_container-#{@post.id}"}
+                - if current_user.liked_posts?(@post)
+                  = render partial: 'partials/unlike', locals: { likable: @post, current_user: current_user}
+                - else
+                  = render partial: 'partials/like', locals: { likable: @post }
+              = render partial: 'partials/like-stats', locals: { likable: @post }
```

Comment:
Same here with the partial. This is the same logic that is used in `app/views/users/show.html.haml`